core/src/test/scala/integration/kafka/api/DescribeAuthorizedOperationsTest.scala

```diff
@@ -73,11 +114,6 @@ class DescribeAuthorizedOperationsTest extends IntegrationTestHarness with SaslS
     TestUtils.waitUntilBrokerMetadataIsPropagated(servers)
   }
 
-  private def accessControlEntry(userName: String, permissionType: AclPermissionType, operation: AclOperation): AccessControlEntry = {
-    new AccessControlEntry(new KafkaPrincipal(KafkaPrincipal.USER_TYPE, userName).toString,
-      AclEntry.WildcardHost, operation, permissionType)
-  }
-
   @AfterEach
   override def tearDown(): Unit = {
     if (client != null)
```

Comment:
unnecessary check

Reply:
I think that it is better to keep it as `client` could not be initialized. For instance, let's say that there is an exception raised when the client is constructed. In this case, `client` would remain `null`.

Reply:
Sorry for unclear comment. 

My point was ```Utils.closeQuietly``` includes null check so we don't need to add same check again.

Reply:
Got it, thanks for the clarification. I have removed the null check.